inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java

```diff
@@ -215,7 +215,7 @@ public Boolean updateTag(ClusterTagRequest request, String operator) {
                     String updateTags = Joiner.on(",").join(tagSet);
                     entity.setClusterTags(updateTags);
                     entity.setModifier(operator);
-                    int rowCount = clusterMapper.updateByIdSelective(entity);
+                    int rowCount = clusterMapper.updateById(entity);
                     if (rowCount != InlongConstants.AFFECTED_ONE_ROW) {
```

Comment:
Personally，local variable 'rowCount' is redundant.

Reply:
done.